pkg/alertmanager/statefulset.go

```diff
@@ -360,15 +360,11 @@ func makeStatefulSetSpec(a *monitoringv1.Alertmanager, config Config, tlsAssetSe
 		"alertmanager":                 a.Name,
 	}
 	if a.Spec.PodMetadata != nil {
-		if a.Spec.PodMetadata.Labels != nil {
-			for k, v := range a.Spec.PodMetadata.Labels {
-				podLabels[k] = v
-			}
+		for k, v := range a.Spec.PodMetadata.Labels {
```

Comment:
I'm just curious if beyond the readability improvement, this change may have some perf impact ~~(probably not expressive enough)~~

Reply:
Owow amazing @Juneezee !
Thanks for sharing that, it was totally out of curiosity hehe